### PR TITLE
New version: OpenMW version 0.48.0

### PR DESCRIPTION
**Pull request by Settyness**
Checklist for Pull Requests
- ✅ Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- ❌ Is there a linked Issue?

Manifests
- ✅ Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- ✅ This PR only modifies one (1) manifest
- ✅ Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- ✅ Have you tested your manifest locally with `winget install --manifest <path>`?
- ✅ Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/157273)

### CODE REVIEW

**Comment by Settyness** [comment]
@microsoft-github-policy-service agree

**Comment by Settyness** [comment]
Not entirely sure what gives with the Validation-Executable-Error label. In my testing it appears to install fine. Whether this caused by the UAC prompt or a missing Visual C++ runtime dependency on older Windows is unclear based on its provided description: https://github.com/microsoft/winget-cli/blob/master/doc/windows/package-manager/package/winget-validation.md

I am testing on 24H2.

openmw-0 48 0-winget-install-success

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Program Files\OpenMW 0.48.0\openmw-bulletobjecttool.exe returned exit code: -1073741515
> Executable C:\Program Files\OpenMW 0.48.0\openmw-cs.exe returned exit code: -1073741515
> Executable C:\Program Files\OpenMW 0.48.0\openmw-essimporter.exe returned exit code: -1073741515
> Executable C:\Program Files\OpenMW 0.48.0\openmw-iniimporter.exe returned exit code: -1073741515
> Executable C:\Program Files\OpenMW 0.48.0\openmw-launcher.exe returned exit code: -1073741515
> Executable C:\Program Files\OpenMW 0.48.0\openmw-navmeshtool.exe returned exit code: -1073741515
> Executable C:\Program Files\OpenMW 0.48.0\openmw-wizard.exe returned exit code: -1073741515
> Executable C:\Program Files\OpenMW 0.48.0\openmw.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 884.)

File: manifests/o/OpenMW/OpenMW/0.48.0/OpenMW.OpenMW.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: OpenMW.OpenMW
+PackageVersion: 0.48.0
+InstallerType: nullsoft
+Installers:
```
Comment (stephengillie) [inline manifests/o/OpenMW/OpenMW/0.48.0/OpenMW.OpenMW.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**